Syntaxes/Chapel.tmLanguage

```diff
@@ -269,7 +269,7 @@
 		<dict>
 			<key>match</key>
 			<string>(?x)\b(
-              	 abs | close | exit| max | min | open | read | readln | sqrt | write | writeln
+              	 abs | close | exit| max | min | open | read | readln | sqrt | write | writeln | writebits | writef | readbits | readf
```

Comment:
Technically none of these are keywords.  I would prefer not to add the bits versions to this list, but am okay with adding writef/readf

Reply:
Then it's fine with leaving them out , rather than unnecessary keywords.